src/System.ComponentModel.Annotations/src/ApiCompatBaseline.net46.txt

```diff
@@ -0,0 +1,2 @@
+MembersMustExist : Member 'System.ComponentModel.DataAnnotations.RegularExpressionAttribute.MatchTimeoutInMilliseconds.get()' does not exist in the implementation but it does exist in the contract.
```

Comment:
These are only available starting in 4.6.1.  This contract only builds a facade for 4.6.1 https://github.com/dotnet/corefx/blob/master/src/System.ComponentModel.Annotations/src/System.ComponentModel.Annotations.builds#L7.  Why is ApiCompat running as 4.6?